scripts/train.py

```diff
@@ -79,6 +79,16 @@ def parse_arguments(self, description, subparser, command):
                             type=int,
                             default=64,
                             help="Batch size, as a power of 2 (64, 128, 256, etc)")
+        parser.add_argument('-l', '--loss-function',
```

Comment:
should the selection of loss function change the default  loss_mix?

Reply:
Currently it doesn't need to. The loss mix is currently only used with (both) the SSIM loss functions, and is set by default to a very low value hardly influencing results - experimentation with this value is needed to see whether SSIM actually improves results or is completely unnecessary.
This was purely added to make experimentation with the Keras.Contrib SSIM implementation possible - I have not had the time to actually test the results.